src/app/core/services/game/hotkeys.ts

```diff
@@ -219,33 +219,49 @@ export class Hotkeys {
       return;
     }
 
-    (this.hotkeyMap[keyCode] || []).forEach(hotkey => {
```

Comment:
this needed to stay an array in case multiple things are bound to a hotkey

changing this has a ton of implications